plugins/strelaysrv/strelaysrv_

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env bash
+: <<=cut
+=head1 NAME
+strelaysrv_ - Plugin to monitor Syncthing relay server
+
+=head1 DESCRIPTION
+This plugin gathers metrics from a Syncthing relay server.
+
+This plugin requires the jq utility : https://stedolan.github.io/jq/
+This plugin requires the curl utility : https://curl.haxx.se/
+
+Available plugins :
+strelaysrv_goroutine #
+strelaysrv_num       # 
+strelaysrv_proxied   #
+strelaysrv_transfer  #
+strelaysrv_uptime    #
+
+=head1 CONFIGURATION
+To make the plugin connect to the Syncthing relay server one has to use this type of 
+configuration
+[strelaysrv_*]
+
+env.STR_HOST 127.0.0.1
+env.STR_PORT 22070
+
+=head1 AUTHOR
+Pierre-Alain TORET <pierre-alain.toret@protonmail.com>
+
+=head1 LICENSE
+MIT
+=cut
+
+function num {
+    case $1 in
+       config)
+            cat <<'EOM'
+graph_title Syncthing relay numbers
+graph_category network
+graph_vlabel numbers
+strelaysrv_num_sessions.label sessions
+strelaysrv_num_connections.label connections
+strelaysrv_num_pending.label pending session keys
+strelaysrv_num_proxies.label proxies
+EOM
+        exit 0;;
+        *)
+            NS=$($CURL -s http://$STR_HOST:$STR_PORT/status | $JQ '.numActiveSessions ')
+            NC=$($CURL -s http://$STR_HOST:$STR_PORT/status | $JQ '.numConnections ')
```

Comment:
Since you are requesting the same resource multiple times, you will slow down the plugin and risk an inconsistent mixture of data points. I would recommend to request the status once, store it in a variable and `echo` it into `jq` for each step.